tests/smoke_tests/run_smoke_test.py

```diff
@@ -182,5 +192,28 @@ async def _wait_for_process_to_finish_and_retrieve_logs(
 
 if __name__ == "__main__":
     loop = asyncio.get_event_loop()
-    loop.run_until_complete(run_smoke_test())
+    loop.run_until_complete(
```

Comment:
Yes, they run sequentially. For paralelization, we need to check the machine requirements to see how many groups of servers-clients it can run and also assign each server to their own port (they are all running on default now). Let me know if you have other concerns besides those.

Reply:
Sounds good. Based on your other response, I think the tests are running in a reasonable amount of time right now. So maybe we look into parallelization later on. Do you mind adding a ticket to clickup?